### ISSUE TITLE
[1.21] Mark 1.21 versions as compatible with 1.21.1

### ISSUE THREAD

**Issue by BugmanBugman** [YUNG-GANG/YUNGs-API#74]
This goes for the other mods too but yeah, you cant download your 1.21 mods on a instance running 1.21.1 becasue it isnt marked as such

**Comment by Timolino456** [comment]
Yes please, I'm using 1.21.1, downloaded a mod from another author that requires YUNG's API but had to download it manually because of this...

**Comment by yungnickyoung** [comment]
Pretty late on this, but this should be fixed now, right?
